bumpversion.sh

```diff
@@ -0,0 +1,47 @@
+current_version="0.4.0"
+new_version=$current_version
+
+__set_new_version()
+{
+  # Split version into major, minor, and patch
+  IFS="." read -a version_array <<< "$current_version"
+  major=$((version_array[0]))
+  minor=$((version_array[1]))
+  patch=$((version_array[2]))
+  # Get version we're bumping from args and change version accordingly
+  case $1 in
+    "major")
+      major=$((major + 1))
+      minor=0
+      patch=0
+      ;;
+    "minor")
+      minor=$((minor + 1))
+      patch=0
+      ;;
+    "patch")
+      patch=$((patch + 1))
+      ;;
+    *)
+      echo "Usage: ./bumpversion.sh major|minor|patch"
+      exit 1
+      ;;
+  esac
+
+  new_version="$major.$minor.$patch"
+}
+
+set -euo pipefail
+
+__set_new_version $1
+
+# Replace only the first instance in Cargo.toml (because there might be others that are for other libraries)
```

Comment:
Question: why not just remove Cargo.toml from the `bumpversion` config file and do that separately?  The rest can still be maintained with a regular bumpversion call.

Reply:
Do you mean calling `bumpversion`for the other files from within that script?  I figured it would be weird to keep the bumpversion dependency if we're already doing something different for one of the files that can be extended to the others.